fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/layers/interning/ScopedInterningLayer.java

```diff
@@ -143,7 +143,7 @@ public static ScopedInterningLayer global(@Nonnull FDBDatabase database) {
         return database.runAsync(context -> interningLayerFuture.thenCompose(layer -> layer.setWindow(context, count)),
                 Lists.newArrayList(
```

Comment:
minor: In theory, this should also be changed to `Arrays.asList` like the other ones that were just changed.

Reply:
I did the minor fix. I was also wrong about the runner, it does log a consistent error of `Retrying FDB Exception`. Maybe not the clearest message in the world, but it is consistent.